UPDATING.md

```diff
@@ -41,6 +41,7 @@ assists people when migrating to a new version.
 
 ### Breaking Changes
 
+- [24686]https://github.com/apache/superset/pull/24686): All dataset's custom explore_url are handled as relative URLs on the frontend, behaviour controlled by PREVENT_UNSAFE_DEFAULT_URLS_ON_DATASET.
```

Comment:
@dpgaspar since we were already validating that all explore_urls were of the same domain, what would possibly break with this change?

Reply:
All `explore_url` is handled as relative, so existing URLs like: `http://www.google.com` will result on the following link: `http://localhost:8088/tablemodelview/list/http://www.google.com`
Previously we were supporting both types, absolute URL and relative, internally when no value was provided we handled the default as relative (dataset link to explore)

Reply:
Is there any way that we can make this somewhat backwards compatible? Maybe a migration that trims off the scheme/domain/host and attempts to make the url into a relative one if it can? Like if .*/explore.* is in the string at least?
Otherwise, we may want to consider this PR for a major version bump. cc @michael-s-molina for thoughts.

Reply:
Yes, I do think this breaks backward compatibility, but users can still opt in for the old behaviour, that I think it's safer and more explicit/clearer then changing metadata with a db migration. 
Ultimately I do question the utility of allowing users to set their own custom default URL for datasets.

@michael-s-molina is it possible to include this one on 3.0.0?

Reply:
Ok, perfect, then, I tagged it for 3.0.